ramls/inventory-hierarchy/inventory-instance-records.json

```diff
@@ -0,0 +1,557 @@
+                }
+              ]
+            },
+            "illPolicy": {
+              "description": "Name of the ILL policy",
+              "type": "string"
+            },
+            "retentionPolicy": {
+              "description": "Records information regarding how long we have agreed to keep something",
+              "type": "string"
+            },
+            "digitizationPolicy": {
+              "description": "Records information regarding digitization aspects",
+              "type": "string"
+            },
+            "holdingsStatements": {
+              "description": "Notes about action, copy, binding etc.",
+              "type": "array",
+              "items": [
+                {
+                  "holdingsNoteTypeName": {
+                    "description": "Name of the holdings note type",
+                    "type": "string"
+                  },
+                  "note": {
+                    "description": "Text content of the note",
+                    "type": "string"
+                  }
+                }
+              ]
+            },
+            "holdingsStatementsForIndexes": {
+              "description": "Holdings record indexes statements",
+              "type": "array",
+              "items": [
+                {
+                  "statement": {
+                    "description": "Textual description of the holdings of indexes",
+                    "type": "string"
+                  },
+                  "note": {
+                    "description": "Note attached to a holdings statement",
+                    "type": "string"
+                  }
+                }
+              ]
+            },
+            "holdingsStatementsForSupplements": {
+              "description": "Holdings record supplements statements",
+              "type": "array",
+              "items": [
+                {
+                  "statement": {
+                    "description": "Textual description of the holdings of supplementary material",
+                    "type": "string"
+                  },
+                  "note": {
+                    "description": "Note attached to a holdings statement",
+                    "type": "string"
+                  }
+                }
+              ]
+            },
+            "copyNumber": {
+              "description": "Piece ID (usually barcode) for systems that do not use holdings record",
+              "type": "string"
+            },
+            "numberOfItems": {
+              "description": "Amount of items of holdings record",
+              "type": "string"
+            },
+            "receivingHistory": {
+              "description": "Receiving history of holdings record",
+              "type": "object",
+              "properties": {
+                "entries": {
+                  "description": "Entries of receiving history",
+                  "type": "array",
+                  "items": [
+                      {
```

Comment:
This fails the build and should be changed to

              "items": {
                "type": "object",
                "properties": {

Reply:
You are right, it looks like I was wrong defined arrays of objects for some entries. I'll fix it, thanks.